ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/traffic-policies.md

```diff
@@ -100,19 +101,36 @@ interface Port-Channel2
 
 | Match set | Type | Sources | Destinations | Protocol | Source Port(s) | Destination port(s) | Action |
 | --------- | ---- | ------- | ------------ | -------- | -------------- | ------------------- | ------ |
-| BLUE-C1-POLICY-01 | ipv4 | 10.0.0.0/8<br/>192.168.0.0/16 | DEMO-01 | tcp | 1,10-20 | ANY | action: PASS<br/>traffic-class: 5 |
+| BLUE-C1-POLICY-01 | ipv4 | 10.0.0.0/8<br/>192.168.0.0/16 | DEMO-01 | tcp<br/>udp | SERVICE-DEMO | ANY | action: PASS<br/>traffic-class: 5 |
```

Comment:
This changes look wrong - where are the TCP ports - they are in the config below?

Reply:
fixed